controllers/image_controller.js

```diff
@@ -73,17 +76,15 @@ async function destroy(req, res) {
   res.json(image);
 }
 
-async function update(req, res) {
-  // let updatedImage = [];
-  // for (let image of req.body.images) {
-  //   updatedImage.push(
-  //     await ImageModel.findByIdAndUpdate(image.id, {
-  //       s3key: image.s3key
-  //     }).catch(err => res.status(500).send(err))
-  //   );
-  // }
-  // res.json(updatedImage);
-  console.log("Update");
+async function update(req, res, next) {
+  console.log(req.body); //Keep it here to see info on console
+
+  await ImageModel.findByIdAndUpdate(req.params.id, {
+    lot: req.body.lot,
+    unitNumber: req.body.unitNumber,
+    productDescription: req.body.productDescription,
+    s3key: req.body.selectedFile // Currently, I do it with a cheeky way. When updating an apartment, the s3key in our database will be updated. But the file is not on AWS S3 hahahahah
```

Comment:
See my comment here. 
Q1: Still need to figure out how to upload file to s3 when users edit apartments & decide to upload a new image
Q2: If users don't upload any image when editing apartment, the app will break. So currently I make it mandatory on frontend (i.e. when users want to edit apartments, they must pick a new image).
Will need to discuss this with you tmr :)

Reply:
ok, thanks